stable/gitlab-ce/Chart.yaml

```diff
@@ -14,6 +15,3 @@ sources:
 - https://hub.docker.com/r/gitlab/gitlab-ce/
 - https://docs.gitlab.com/omnibus/
 icon: https://gitlab.com/uploads/group/avatar/6543/gitlab-logo-square.png
-maintainers:
-- name: Greg Taylor
```

Comment:
why remove maintainers? instead of the name it should be the id though, which is: gtaylor

Reply:
doh - I didn't see the above answer - disregard this

Reply:
@davidkarlsen the linter requires it, I don't know why. See: https://github.com/helm/charts/pull/7614#issuecomment-419567032